kata-deploy/scripts/kata-deploy.sh

```diff
@@ -142,29 +142,29 @@ function configure_containerd() {
 		cp "$containerd_conf_file" "$containerd_conf_file_backup"
 	fi
 	cat <<EOT | tee -a "$containerd_conf_file"
-[[plugins]]
-  [[plugins.cri]]
-   [[plugins.cri.containerd]]
-     [plugins.cri.containerd.runtimes.kata]
+# [plugins]
+  # [plugins.cri]
+    # [plugins.cri.containerd]
```

Comment:
[Some containerd config files](https://github.com/containerd/containerd/blob/master/docs/man/containerd-config.toml.5.md#example) indent for nesting and [others don't](https://github.com/rancher/k3s/blob/master/pkg/agent/templates/templates.go). ([Indentation is optional in TOML](https://github.com/toml-lang/toml#example))

I went with indenting but including commented out super-tables to prevent the rest from seemingly floating to the right. Alternatively, super-tables could be completely removed and the rest not indented.